cpp/ql/lib/qlpack.yml

```diff
@@ -7,3 +7,4 @@ library: true
 upgrades: upgrades
 dependencies:
   codeql/ssa: ${workspace}
+  codeql/tutorial: ${workspace}
```

Comment:
What is the reason for adding a dependency on the tutorial pack? The pack is unused.

Reply:
Such that you can do `import tutorial` from any file.